mods/ts/rules/defaults.yaml

```diff
@@ -887,9 +887,11 @@
 		DamageInterval: 16
 		DamageTypes: BulletDeath
 		Terrain: Veins
-	GrantConditionOnTerrain@VEINS:
+	LeavesTrails@VEINS:
+		Image: veins
+		Palette: player
 		TerrainTypes: Veins
-		Condition: veins
-	WithIdleOverlay@VEINS:
-		Sequence: veins
-		RequiresCondition: veins
+		TrailWhileStationary: true
+		StationaryInterval: 16
+		MovingInterval: 16
```

Comment:
This may be a bit long for faster vehicles like the attack buggy?  It will travel multiple cells before the animation repeats.